lib/tasks/temporary/measure_feeds.rake

```diff
@@ -0,0 +1,96 @@
+# Temporary tool used to have a rough idea of timings and memory occupation
+require "memory_profiler"
+
+class FeedsImportMeasurer
+  class << self
+    def rss_reader(args)
+      call(args, :rss_reader)
+    end
+
+    def feeds_import(args)
+      call(args, :feeds_import)
+    end
+
+    def call(args, importer)
+      setup_feeds(num_users: args[:num_users])
+
+      memory_report = MemoryProfiler.report do
+        speed_report = Benchmark.measure do
+          puts "Importing #{num_users} feeds with #{importer}..."
+
+          case importer
+          when :rss_reader
+            RssReader.get_all_articles
+          when :feeds_import
+            Feeds::Import.call
+          else
+            raise ArgumentError, "unknown importer: #{importer}"
+          end
+        end
+
+        puts speed_report
+      end
+
+      filepath = "/tmp/measure_feeds_#{importer}_#{num_users}_#{Time.current.iso8601}.txt"
+      puts "Saving memory profile to #{filepath}..."
+      puts memory_report.pretty_print(to_file: filepath)
+    end
+
+    def setup_feeds(num_users: nil)
+      # we start from no articles in the DB
+      Article.delete_all
```

Comment:
Can we remove this from this PR and keep it in a gist or something since it's temporary? I dont want code like this going out to production in the event for some reason it is accidentally triggered.

Reply:
Makes sense! I'll move the script :)